### DISCUSSION TITLE
v0.16.2

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1159] [Releases]
## What's Changed
* Hotfix v0.16.1 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1156
* Hotfix v0.16.2 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1158

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.16.1...v0.16.2

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.16.2'>v0.16.2</a>.</em>

**Comment by zackmuc** [comment]
It still shows 16.1 and the result page does not work ,

**Reply by alexjustesen** [reply]
Ffs, forgot to update the build info

**Reply by alexjustesen** [reply]
0.16.3 I'll deploy after dinner

**Comment by RobertD502** [comment]
Couple things I noticed:

1. The results page (after migration) isn't showing the correct values for down/up while the dashboard shows the correct values.

Example:

![image](https://github.com/alexjustesen/speedtest-tracker/assets/52541649/548b292c-d562-4424-83b7-33ecd3e1b014)
That should be a down of `562.13` and an upload of `16.02`.

The speedtest result page linked to the speedtest shown above:

![image](https://github.com/alexjustesen/speedtest-tracker/assets/52541649/99271589-f4c6-4086-933a-803f48b5cf76)

2. The API is not returning the speed results in Mbps. It is instead returning them as bits per second. This causes issues in dashboards and outside automations where Mbps is expected (since this is what the API previously returned).

![image](https://github.com/alexjustesen/speedtest-tracker/assets/52541649/7bdebd15-2a43-40da-a7f4-6ffb6a75a68b)

**Reply by skyeci-draytek** [reply]
same here for me - results page displaying incorrect values.

**Reply by alexjustesen** [reply]
1. Already fixed in #1160 👍
2. Good catch, can you open an issue if one doesn't already exist?

**Reply by masterwishx** [reply]
Same for me

**Comment by masterwishx** [comment]
No info in influxDBv2 passed

**Comment by masterwishx** [comment]
![image](https://github.com/alexjustesen/speedtest-tracker/assets/28630321/b559db87-cdab-48a8-8fca-78598bbbe02b)

![image](https://github.com/alexjustesen/speedtest-tracker/assets/28630321/abb02422-79d9-4d20-9ce8-21dbc8355e24)

**Reply by masterwishx** [reply]
![image](https://github.com/alexjustesen/speedtest-tracker/assets/28630321/34406863-d5c6-444b-9fff-852bf5061439)

**Comment by masterwishx** [comment]


**Reply by RobertD502** [reply]
@masterwishx Add the images that you just uploaded as replies to your original post - this will keep things organized within a single thread/allow discussion specific to the issue you are facing to be centralized in one place.
